worker/orgfetch.js

```diff
@@ -31,6 +35,57 @@ async function updateChildrenFromParents(job) {
 	}
 }
 
+async function updateOrgLocation(job) {
+	adminJob = job;
+
+	try {
+		let query = ` SELECT DISTINCT ON (instance) instance AS key, org_location, org_env, org_release, org_status FROM org `;
+		let instances_from_db = await db.query(query);
```

Comment:
Code style => no underscores in variable names.  underscores only appropriate for constants w/ all caps.